sql/2020/17_JAMstack/cdn_core_web_vitals_distribution.sql

```diff
@@ -0,0 +1,64 @@
+#standardSQL
+# Core Web Vitals distribution by CDN
+#
+# Note that this is an unweighted average of all sites per SSG.
+# Performance of sites with millions of visitors as weighted the same as small sites.
+SELECT
+  client,
+  CDN,
+  COUNT(DISTINCT origin) AS origins,
+  SUM(fast_lcp) / (SUM(fast_lcp) + SUM(avg_lcp) + SUM(slow_lcp)) AS good_lcp,
```

Comment:
(Throughout) It might not matter but this is simpler and consistent with other CWV queries
```suggestion
  SUM(fast_lcp) / (SUM(fast_lcp + avg_lcpslow_lcp)) AS good_lcp,
```

Reply:
This comment applies throughout the query